src/components/TodoInfo/TodoInfo.scss

```diff
@@ -0,0 +1,17 @@
+.TodoInfo {
+  padding: 8px;
+  margin: 12px 0;
+  border: 1px solid #000;
+  border-radius: 8px;
+  background-color: antiquewhite;
+
+  &__title {
+    margin: 4px 0;
+    font-size: 28px;
+    color: #f00;
```

Comment:
Consider using a more readable color for the title. For example, `#c00` instead of `#f00`. 🎨